lib/inspec/resources/opa_api.rb

```diff
@@ -0,0 +1,41 @@
+require "inspec/resources/opa"
+
+module Inspec::Resources
+  class OpaApi < Opa
+    name "opa_api"
+    supports platform: "unix"
+    supports platform: "windows"
+
+    attr_reader :allow
+
+    def initialize(opts={})
+      @url = opts[:url]
+      @data = opts[:data]
+      fail_resource "policy and data are the mandatory for executing OPA." if @url.nil? && @data.nil?
```

Comment:
I think this should be an `||`, and also you should update the message to be "url and data are mandatory", not "policy and data".